<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QWidget" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>输入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtIn">
       <property name="text">
        <string>hello</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAppend">
       <property name="text">
        <string>Append</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnThread">
       <property name="text">
        <string>start thread</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTimer">
       <property name="text">
        <string>start timer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>输出</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="txtOut"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
